spec/models/inventory_food_spec.rb

```diff
@@ -0,0 +1,70 @@
+# == Schema Information
+#
+# Table name: inventory_foods
+#
+#  id           :bigint           not null, primary key
+#  quantity     :integer
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  food_id      :bigint           not null
+#  inventory_id :bigint           not null
+#
+# Indexes
+#
+#  index_inventory_foods_on_food_id       (food_id)
+#  index_inventory_foods_on_inventory_id  (inventory_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (food_id => foods.id)
+#  fk_rails_...  (inventory_id => inventories.id)
+#
+require 'rails_helper'
+
+RSpec.describe InventoryFood, type: :model do
+  let(:user) do
+    FactoryBot.create(:user, name: 'Edward Yara')
+  end
```

Comment:
- [ ] Kindly update the user creation to include `email`, `password`, `password_confirm` fields since the models spec can't pass without them.